static/app/views/issueDetails/streamline/solutionsHubDrawer.tsx

```diff
@@ -14,17 +15,26 @@ import {useAutofixSetup} from 'sentry/components/events/autofix/useAutofixSetup'
 import {DrawerBody, DrawerHeader} from 'sentry/components/globalDrawer/components';
 import {GroupSummaryBody, useGroupSummary} from 'sentry/components/group/groupSummary';
 import Input from 'sentry/components/input';
-import {IconSeer} from 'sentry/icons';
-import {t} from 'sentry/locale';
+import {IconDocs, IconSeer} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import type {Event} from 'sentry/types/event';
+import {EntryType, type Event} from 'sentry/types/event';
 import type {Group} from 'sentry/types/group';
+import type {Organization} from 'sentry/types/organization';
 import type {Project} from 'sentry/types/project';
 import {getShortEventId} from 'sentry/utils/events';
+import {
+  getConfigForIssueType,
+  shouldShowCustomErrorResourceConfig,
+} from 'sentry/utils/issueTypeConfig';
+import {getRegionDataFromOrganization} from 'sentry/utils/regions';
 import useRouteAnalyticsParams from 'sentry/utils/routeAnalytics/useRouteAnalyticsParams';
+import useOrganization from 'sentry/utils/useOrganization';
 import {MIN_NAV_HEIGHT} from 'sentry/views/issueDetails/streamline/eventTitle';
+import Resources from 'sentry/views/issueDetails/streamline/resources';
+import {useIsSampleEvent} from 'sentry/views/issueDetails/utils';
 
-import {AutofixSetupContent} from './autofixSetupModal';
+import {AutofixSetupContent} from '../../../components/events/autofix/autofixSetupModal';
```

Comment:
```suggestion
import {AutofixSetupContent} from 'sentry/components/events/autofix/autofixSetupModal';
```
Does a root import work here?